proto/shared_constants.proto

```diff
@@ -93,6 +93,10 @@ message DVAState {
 
     // infracted
     infracted = 5;
+
+    // operational
+    // TODO.
```

Comment:
NIT: Might need to be removed?

Reply:
I left the TODO in there so we can remove static_operational and dynamic_operational in the future